index.bs

```diff
@@ -8215,85 +8370,102 @@ for the specific requirements that the use of
+            1.  Return |defaults|.
 
-    <dl class="switch">
-         :  interface
-         :  namespace
-         :: The [=exposure set=] of the
-            interface or namespace only contains the
-            [=primary global interface=].
-         :  partial interface
-         :  partial namespace
-         :: The [=exposure set=] of the partial
-            interface or namespace is the [=exposure set=] of the original interface or namespace definition.
-         :  interface member
-         :: The [=exposure set=] of the interface member
-            is the [=exposure set=] of the interface or
-            partial interface the member is declared on.
-         :  namespace member
-         :: The [=exposure set=] of the
-            namespace member is the [=exposure set=] of the namespace or partial namespace the member is declared on.
-    </dl>
+</div>
 
-If [{{Exposed}}] appears on an
-[=overloaded=] [=operation=],
+If [{{Exposed}}] appears on an [=overloaded=] [=operation=],
 then it must appear identically on all overloads.
 
-The [{{Exposed}}] extended attribute
-must not be specified on both an interface
-member and a partial interface definition the interface member is declared on.
-Similarly, the [{{Exposed}}] extended attribute
-must not be specified on both a namespace
-member and a partial namespace definition the namespace member is declared on.
-
-If [{{Exposed}}] appears an interface member, then
-the interface member's [=exposure set=]
-must be a subset of the [=exposure set=] of the interface or partial interface it's a
-member of. Similarly, if [{{Exposed}}] appears on a
-namespace member, then the namespace member's [=exposure set=] must be a
-subset of the [=exposure set=] of the
-namespace or partial namespace it's a member of.
-
-An interface's [=exposure set=]
-must be a subset of the
-[=exposure set=] of all
-of the interface's [=consequential interfaces=].
+The [{{Exposed}}] extended attribute must not be specified both on
+an [=interface member|interface=], [=mixin member|mixin=], or [=namespace member=], and on
+the [=partial interface=], [=partial mixin=], or [=partial namespace=] definition
+the [=member=] is declared on.
+
+Note: This is because [{{Exposed}}] extended attribute on partials are basicaly shorthand
+for annotating every [=member=] of the partial.
+
+If [{{Exposed}}] appears on a [=partial interface=] or [=partial namespace=],
+then the partial's [=own exposure set=] must be a subset of
+the [=exposure set=] of the partial's original [=interface=] or [=namespace=].
+
+If [{{Exposed}}] appears on an [=interface member|interface=] or [=namespace member=],
+then the [=member=]'s [=exposure set=] must be a subset
+of the [=exposure set=] of the [=interface=] or [=namespace=] it is a member of.
+
+If [{{Exposed}}] appears both on a [=partial mixin=] and its original [=mixin=],
+then the [=partial mixin=]'s [=own exposure set=]
+must be a subset of the [=mixin=]'s [=own exposure set=].
```

Comment:
Hmm, why "own" for mixin partials but not-own for interface and namespace partials? I imagine you've thought about this longer so I'm sure you've got it right, I'm just hoping you can explain it to me :)

Reply:
For two reasons:

1. Mixins can't have an exposure set as they're just a container for mixin members that melts away when included in an interface, just like a document fragment does when added to a document. They's need to have multiple exposure sets (corresponding to all of the interfaces they get included in), or they'd need to be copied whenever they're included somewhere. So far it seemed like a better mental model to only create copies of mixin members when a mixin is included, and not also of the mixin itself (nor its partial). That prevents us from giving mixins an exposure set, which I think is actually a good thing; their editorial aides, not constructs designed to have JS bindings and show up in the JS environment. 
2. For namespaces and interfaces, however, we can't rely solely on their "own exposure set"; we need to consider the case where they have none and need to default to « primary global ». This will no longer be an issue once we merge #423.